node/full_node_integration_test.go

```diff
@@ -150,7 +150,10 @@ func TestLazyAggregator(t *testing.T) {
 	key, _, _ := crypto.GenerateEd25519Key(rand.Reader)
 	genesisValidators, signingKey := getGenesisValidatorSetWithSigner(1)
 	blockManagerConfig := config.BlockManagerConfig{
-		BlockTime:   1 * time.Millisecond,
+		// requires min 150ms between the genesis header time and
+		// syncer to find the genesis header, otherwise it will
+		// skip it as not recent
+		BlockTime:   350 * time.Millisecond,
```

Comment:
if this becomes a bottleneck in the future in terms of CI speed, we could look into build variables to reduce the 150ms genesis header time potentially. Just a thought for the future.